.github/workflows/aws-runner-template.yaml

```
@@ -24,12 +22,13 @@ jobs:
 
       - name: Start EC2 runner
         id: start-ec2-runner
-        uses: machulav/ec2-github-runner@v2
+        uses: machulav/ec2-github-runner@28fbe1c4d7d9ba74134ca5ebc559d5b0a989a856
```

I think we can use the tag, which might be more convenient
```suggestion
        uses: machulav/ec2-github-runner@v2.3.8
```
Here is an example run that worked in my fork: https://github.com/tverghis/memory-collector/actions/runs/12922155530

---

great suggestion! much more readable this way. ty